app/controllers/weets.js

```diff
@@ -0,0 +1,21 @@
+const userService = require('../services/users');
+const jokesService = require('../services/jokes');
+const weetService = require('../services/weets');
+const logger = require('../logger');
+
+exports.postWeets = async (req, res, next) => {
+  try {
+    const user = req.body;
+    const userFound = await userService.findUser(user);
+    if (userFound) {
```

Comment:
It is not necessary to send user information (email), because we can obtain it from token.